contracts/implementations/tableland/DeviceDefinitionTable.sol

```diff
@@ -372,4 +469,46 @@ contract DeviceDefinitionTable is AccessControlInternal {
 
         return concatenatedData;
     }
+
+    function _updateDeviceDefinitionData(
+        TablelandTablesImpl tablelandTables,
+        uint256 tableId,
+        DeviceDefinitionInput calldata data
```

Comment:
Looks like we just want to modify the metadata here. So we can save some gas just passing the `id` and `metadata` as specific params instead of the whole object.

Reply:
The function signature was changed.